pkg/trace/info/stats.go

```diff
@@ -198,7 +308,28 @@ type Stats struct {
 
 func (s *Stats) update(recent *Stats) {
 	atomic.AddInt64(&s.TracesReceived, atomic.LoadInt64(&recent.TracesReceived))
-	atomic.AddInt64(&s.TracesDropped, atomic.LoadInt64(&recent.TracesDropped))
+
+	// TracesDropped
```

Comment:
Do we really need these comments? Do they add anything?

Reply:
Removed. Was just following comment style I saw elsewhere in this code